src/burn/engine/plan.cpp

```diff
@@ -1033,6 +1037,9 @@ extern "C" HRESULT PlanCachePackage(
 
     if (fBARequestedCache || NeedsCache(pPlan, pPackage))
     {
+        // The behavior for cache only mode is to do nothing on rollback, e.g. for subsequent install on demand scenarios.
```

Comment:
A bundle can remove the cache if it's not installed, but the user can't run the bundle unless it's available somewhere.  It's not very user friendly for a package to be put in the package cache without also ensuring its owning bundle is also in the cache.